example-app/src/admin/resources/create-resource-result.type.ts

```diff
@@ -3,5 +3,5 @@ import { ResourceOptions, FeatureType } from 'admin-bro'
 export type CreateResourceResult<T> = {
   resource: T;
   options: ResourceOptions;
-  features: [FeatureType];
+  features: Array<FeatureType>;
```

Comment:
What is the purpose of this change? Is there any other difference than syntax?

Reply:
no - they are different. `[FeaureType]` is a `Tuple` with the actual length, where `Array<FeatureType>` is an regular `Array`. The first one returns an error when you want to add more than one feature